app/client/views/settings/data/data.html

```diff
@@ -12,22 +12,118 @@ <h1>
+          ><i class="fa fa-2x fa-database" aria-hidden="true"></i
+        ></span>
+      </div>
+      <!-- /.box-header -->
 
       <div class="box-body">
         <p>
           {{_ "dataSettings-rawData-helpText" }}
         </p>
-        <button type="button" class="btn btn-success" id="export-data" disabled={{ fetchingData }}>
+        <button
+          type="button"
+          class="btn btn-success"
+          id="export-data"
+          disabled="{{ fetchingData }}"
+        >
           <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>&nbsp;
           {{_ "dataSettings-rawData-exportRawDataButton" }}
         </button>
 
         {{# if showSpinner }}
-          {{> spinner }}
-        {{/ if }}
-      </div><!-- /.box-body -->
-    </div><!-- /.box -->
+        {{> spinner }}
+        {{/if }}
+      </div>
+      <!-- /.box-body -->
+    </div>
+    <!-- /.box -->
+
+    <div class="box box-primary">
+      <div class="box-header with-border">
+        <h3 class="box-title">
+          {{_ "dataSettings-rawData-import-header" }}
+        </h3>
+      </div>
+      <!-- /.box-header -->
+
+      <div class="box-body">
+        <p>
+          {{_ "dataSettings-rawData-import-helpText" }}
+        </p>
+
+        {{# if shouldShowAlert}}
+        <div class="alert alert-danger pull-right col-xs-4" role="alert">
+          {{_ "dataSettings-invalid-json"}}
+        </div>
+        {{/if}}
+
+        {{#if isRespError}}
+        <div
+          class="alert alert-danger alert-dismissible pull-right col-xs-4"
+          role="alert"
+        >
+          <button
+            type="button"
+            class="close"
+            id="closeImportErrAlert"
+            data-dismiss="alert"
+            aria-label="Close"
+          >
+            <span aria-hidden="true">&times;</span>
+          </button>
+          {{ respMessage }}
+        </div>
+
+        {{/ if}}
+
+        <div class="container">
+          <div class="row">
+            <input
+              id="inputFile"
+              accept="application/json"
+              style="display: none;"
+              class="file-upload-input file-upload"
+              type="file"
+            />
+            <a
+              ><label for="inputFile" class="no-margin"
+                >{{_ "dataSettings-clickToBrowse"}}</label
+              ></a
+            >
+            {{# if isDisabled  }}
+            <button type="button" id="importData" class="btn btn-primary">
+              {{_ "dataSettings-importDataBtn"}}
+            </button>
+            {{else}}
+            <button
+              disabled
+              type="button"
+              id="importData"
+              class="btn btn-primary"
+            >
+              {{_ "dataSettings-importDataBtn"}}
+            </button>
+            {{/ if}}
+            {{# if showModal}}
+            {{> importSummary data=importRes}}
+            {{/if}}
+          </div>
+          {{# if ifFileExists file}}
```

Comment:
`{{# if file }}` should be sufficient here, without the `ifFileExists` function. Let me know if that is not the case.

Reply:
Yes, it should be, But I was going for an expression here so that it always evaluates to a boolean value

Reply:
That makes sense.

If the `{{# if file }}` approach works without concern for ambiguous bugs, and is close to how it would be said in *plain language*, lets just do without the `ifFileExists` function. 

Fewer moving parts and the same level of readability.